doc/scalar/riscv-crypto-scalar-entropy-source.adoc

```diff
@@ -246,16 +246,18 @@ The `seed` CSR is by default only available in M mode, but can be made
 available to other modes via the `mseccfg.sseed` and `mseccfg.useed`
 access control bits. `sseed` is bit `9` of and `useed` is
 bit `8` of the `mseccfg` CSR.
-Without the corresponding access control bit set to 1, an attempted
-read/write access to `seed` from U, S, or HS modes will raise an
-illegal instruction Exception. 
-
-Attempted access to `seed` from virtual modes VS and VU always raises an
-exception; a read-only instruction causes an illegal instruction Exception,
-while a read-write instruction (that can potentially be emulated) causes
-a virtual instruction Exception. Note that HS, VS, and VU modes are
-present in systems with Hypervisor (H) extension implemented. If desired, 
-a hypervisor can emulate accesses to the seed CSR from a virtual machine.
+Without the corresponding access control bit set to 1, any attempted
+access to `seed` from U, S, or HS modes will raise an illegal instruction
+exception. 
+
+VS and VU modes are present in systems with Hypervisor (H) extension
+implemented. If desired, a hypervisor can emulate accesses to the seed CSR
+from a virtual machine. Attempted access to `seed` from virtual modes
+VS and VU always raises an exception; a read-only instruction causes an
+illegal instruction exception, while a read-write instruction (that can
+potentially be emulated) causes a virtual instruction exception only if
+`mseccfg.seed=1`. Note that `mseccfg.useed` has no effect on the exception
```

Comment:
`mseccfg.seed=1` should be `mseccfg.sseed=1`, no?

Reply:
Yes, of course. Fixed now I think.